.github/shared/src/swagger.js

```diff
@@ -118,6 +150,61 @@ export class Swagger {
     return filtered;
   }
 
+  /**
+   * @returns {Promise<Operation[]>}
+   */
+  async getOperations() {
+    if (this.#operations.length === 0) {
+      try {
+        const content = await readFile(this.#path, "utf8");
+        const swagger = JSON.parse(content);
+        // Process regular paths
+        if (swagger.paths) {
+          for (const [path, pathItem] of Object.entries(swagger.paths)) {
+            for (const [method, operation] of Object.entries(pathItem)) {
+              if (
+                typeof operation === "object" &&
+                operation.operationId &&
+                method !== "parameters"
+              ) {
+                this.#operations.push({
+                  id: operation.operationId,
+                  httpMethod: method.toUpperCase(),
+                  path: path,
+                });
+              }
+            }
+          }
+        }
+
+        // Process x-ms-paths (Azure extension)
+        if (swagger["x-ms-paths"]) {
+          for (const [path, pathItem] of Object.entries(swagger["x-ms-paths"])) {
+            for (const [method, operation] of Object.entries(pathItem)) {
+              if (
+                typeof operation === "object" &&
+                operation.operationId &&
+                method !== "parameters"
+              ) {
+                this.#operations.push({
+                  id: operation.operationId,
+                  httpMethod: method.toUpperCase(),
+                  path: path,
+                });
+              }
+            }
+          }
+        }
+
+        // Sort operations by id and store in instance variable
+        this.#operations.sort((a, b) => a.id.localeCompare(b.id));
+      } catch {
+        throw new SpecModelError(`Failed to read operations from swagger: ${this.#path}`);
+      }
```

Comment:
```suggestion
      } catch {
        throw new SpecModelError(`Failed to read operations from swagger: ${this.#path}`);
      }
```

Just let the exception bubble.

Reply:
It already does.